ci/cold_96_00z.yaml

```
@@ -0,0 +1,20 @@
+environment:
+  rotdir: ${RUNTEST}/cold_96_00z
+  HOMEgfs: ${HOMEGFS}   #TODO - currenlty using the env var ${HOMEgfs}
+                        #cause XML file to have HOMEgfs set to None 
+experment:
+  mode: cycled
+
+arguments:  
+   pslot: cold_96_00z  #TODO pslot must be the same as above in rotdir 
+   app: ATM            #this sould be updated when forward stringfication
+   resdet: 96          #is supported in the YAMLFile class
+   resens: 48
+   comrot: ${RUNTEST}
+   expdir: ${RUNTEST}/expdir
+   icsdir: /scratch1/NCEPDEV/global/glopara/data/ICSDIR/C96C48
```

should probably do something like:
```suggestion
   icsdir: ${ICSDIR_ROOT}/C96C48
```
and then define `ICSDIR_ROOT` in `${machine}.sh`
I think it will help with portability of the test.

---

Yes,  Walter noted this too, that should be updated now then, will do.